doc/src/reference/py.rst

```diff
@@ -121,6 +121,15 @@ argument
 
     :rtype: argument
 
+.. py:function:: create_argument(s, values)
+
+    Create an argument of shape s with a set of values.
+
+    :param shape s: Shape of argument to create.
+    :param list values: Values to put in the argument. Must be the same number of elements as the shape.
```

Comment:
> Must be the same number of elements as the shape

This is not enforced. Assert inside `fill()` checks for `<=`.

Reply:
Added a check for the size.